client/data/promote-post/use-promote-post-campaigns-query-new.ts

```diff
@@ -33,6 +33,7 @@ export type CampaignResponse = {
 		total_budget_used: number;
 		display_delivery_estimate: string;
 		views_total: number;
+		stats_enabled: number;
```

Comment:
I think we return this as boolean in the DSP.

Reply:
whoops, changed